tests/config/functional/toml/unknown_msgid/enable_unknown_msgid.result.json

```diff
@@ -0,0 +1,9 @@
+{
+  "functional_append": {
+    "disable": ["logging-format-interpolation"],
+    "enable": ["locally-disabled"]
```

Comment:
I don't know if it's a good thing that the issue is silently ignored when we do not recognize a msgid or symbol but this is what happens right now.

Reply:
Maybe something to handle in another PR.

Reply:
Yeah this what was I was talking about in that previous PR. There is an `TODO: Optparse` for it!